optimum/amd/cli.py

```diff
@@ -1,47 +1,111 @@
 import os
+import subprocess
 import sys
-from argparse import ArgumentParser
 
-from .topology_utils import extract_max_avg_bandwidth_cluster, get_bandwidth_matrix
 
+try:
```

Comment:
All of this could be worth moving to Accelerate no? They have their own `accelerate launch` which is more widely used & advertised in Transformers doc/examples.

Reply:
yes it makes sense for distributed inference/training but not sure about zentorch environment variables.

Reply:
why not?

Reply:
I thought `launch` can only be used in a distributed setting, but looking at the docs I guess not necessarily.